apps/site/pages/components/organisms/newsletter.mdx

```diff
@@ -0,0 +1,385 @@
+---
+description: The Newsletter component is a section with at least one input text with an action which its value could be submitted..
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Hero___c73fdfe80ce87c2ccc4701c07e9d5398.png
+---
+
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+
+import NewsletterUsage from 'site/components/Newsletter/NewsletterUsage'
+
+import path from 'path'
+import { useSSG } from 'nextra/ssg'
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const newsletterPath = path.resolve(__filename)
+  const components = [
+    'Newsletter.tsx',
+    'NewsletterAddendum.tsx',
+    'NewsletterContent.tsx',
+    'NewsletterForm.tsx',
+    'NewsletterHeader.tsx',
+  ]
+  const [
+    newsletterProps,
+    newsletterAddendumProps,
+    newsletterContentProps,
+    newsletterFormProps,
+    newsletterHeaderProps,
+  ] = getComponentPropsFrom(newsletterPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: {
+        newsletterProps,
+        newsletterAddendumProps,
+        newsletterContentProps,
+        newsletterFormProps,
+        newsletterHeaderProps,
+      },
+    },
+  }
+}
+
+export const NewsletterPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const {
+    newsletterProps,
+    newsletterAddendumProps,
+    newsletterContentProps,
+    newsletterFormProps,
+    newsletterHeaderProps,
+  } = useSSG()
+  return {
+    Newsletter: <PropsSection propsList={newsletterProps} />,
+    NewsletterAddendum: <PropsSection propsList={newsletterAddendumProps} />,
+    NewsletterContent: <PropsSection propsList={newsletterContentProps} />,
+    NewsletterForm: <PropsSection propsList={newsletterFormProps} />,
+    NewsletterHeader: <PropsSection propsList={newsletterHeaderProps} />,
+  }[component]
+}
+
+<header>
+
+# Newsletter
+
+The Newsletter component is a section with at least one input text with an action which its value could be submitted.
+
+</header>
+
+## Import
+
+Import the components from [@faststore/ui](/../components)
+
+```tsx
+import {
+  Newsletter,
+  NewsletterAddendum,
+  NewsletterContent,
+  NewsletterForm,
+  NewsletterHeader,
+} from '@faststore/ui'
+```
+
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/organisms/Newsletter/styles.scss'
+```
```

Comment:
```suggestion
### Import Styles into your FastStore project

To apply the styles of this component in your FastStore project, import the following into your stylesheet:

```scss copy
@import '@faststore/ui/src/components/organisms/Newsletter/styles.scss';
```

Follow the instructions in the [Importing FastStore UI component styles](/docs/customizing/importing-styles-to-faststore-ui-components) tutorial.
```

Reply:
Updating to our new format. (The suggestion tool is not working as expected here), but here is the suggestion:

### Import Styles into your FastStore project

To apply the styles of this component in your FastStore project, import the following into your stylesheet:

```scss copy
@import '@faststore/ui/src/components/organisms/Newsletter/styles.scss';
```

Follow the instructions in the [Importing FastStore UI component styles](/docs/customizing/importing-styles-to-faststore-ui-components) tutorial.